src/index.ts

```diff
@@ -60,6 +60,10 @@ export interface Options {
    * Generate unknown type instead of any
    */
   unknownAny: boolean
+  /**
+   * JSDoc tags to parse from the schema. Will be prepended to each generated type in the comment block (as `@tag [tagValue]`).
+   */ 
+  jsdocTags: string[]
```

Comment:
Instead of making this a configurable option, what do you think of emitting an `@tag [tagValue]` line for each property that's a valid JSONSchema property, but not supported by JSON Schema to TypeScript? We know what these are, so there's no need to make consumers manually pass it in.

*utils.ts* would be a good place to put a `Set` of those properties, which you can then check against in `parseTags`.

Reply:
Hmm... In fact I'm using this library to generate a custom JSON (VSCode contribution points json), not specifically a JSON Schema.
So, I agree that having default values for known JSON Schema properties is a good idea, but I still want to have a way to generate from any custom tag. 
Does that make sense?